@here/harp-mapview/lib/geometry/TileGeometryCreator.ts

```diff
@@ -686,18 +686,6 @@ export class TileGeometryCreator {
                     const lineMaterial = material as THREE.RawShaderMaterial;
                     lineMaterial.uniforms.opacity.value = material.opacity;
 
-                    if (
```

Comment:
Why do you completely disable clipping? This might introduce other artifacts with transparent lines in tiles with a buffer area...